libs/logging-bundle/tests/KeboolaLoggingBundleTestingKernel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\LoggingBundle\Tests;
+
+use Keboola\LoggingBundle\KeboolaLoggingBundle;
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\MonologBundle\MonologBundle;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+class KeboolaLoggingBundleTestingKernel extends Kernel
```

Comment:
Doufám, že tohle je kompatibilní i se symfony 6, ať to pak neupdatujeme všechno nadvakrát.

Reply:
cert vi, to se pozna :smile: ale uz vymyslim, jak to (jednoduse) testovat na vice verzich Symfony